tests/conftest.py

```diff
@@ -232,7 +205,7 @@ def localhost(ansible_adhoc):
 
 
 @pytest.fixture(scope="module")
-def ptfhost(ansible_adhoc, testbed):
+def ptfhost(ansible_adhoc, testbed, duthost):
```

Comment:
is this a miss?

Reply:
Yes. Calling fixture duthost was missed in another PR https://github.com/Azure/sonic-mgmt/pull/1658. So I added it here.